test/files/pos/t6734.scala

```diff
@@ -0,0 +1,23 @@
+
+//single file badimp.scala
+// adding package object gives not found: type SortedMap
+package object badimp
```

Comment:
How about:

```
// this package object will contain the synthetic companion object of `Nodal` and fail because the 
// necessary import is not in this scope